csharp/Directory.Build.props

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <!-- Common repo directories -->
+  <PropertyGroup>
+    <RepoRoot>$(MSBuildThisFileDirectory)../</RepoRoot>
+    <CSharpDir>$(MSBuildThisFileDirectory)</CSharpDir>
+    <BaseOutputPath>$(CSharpDir)/artifacts/$(MSBuildProjectName)</BaseOutputPath>
+  </PropertyGroup>
+
+  <!-- AssemblyInfo properties -->
+  <PropertyGroup>
+    <Product>Apache Arrow library</Product>
+    <Copyright>2018 Apache Software Foundation</Copyright>
+    <Company>Apache</Company>
```

Comment:
"Apache Software Foundation" may be better.
If we need to change this, we can work on this as a follow-up task.

Reply:
yes the formal name of the organization is "The Apache Software Foundation"

Reply:
OK.
@eerhardt Could you create a pull request for this?

Reply:
Will do. Can I use the same JIRA?

Reply:
It can be a simple PR without a JIRA since it's not notable for the changelog